mock/ereg/virksomhetMock.ts

```diff
@@ -5,8 +5,11 @@ import {
   VIRKSOMHET_PONTYPANDY,
   VIRKSOMHET_UTEN_NARMESTE_LEDER,
 } from "../common/mockConstants";
+import { EregOrganisasjonResponseDTO } from "@/data/virksomhet/types/EregOrganisasjonResponseDTO";
```

Comment:
Vi har litt trøbbel med disse import-aliasene i mockene så tror det må endres til `../../src/data/virksomhet/types/EregOrganisasjonResponseDTO `for at man ikke skal få feil ved oppstart lokalt. Gjelder de andre mock-filene også. Tar gjerne mot forslag til løsning :)